File: Codechef/Hire_Test/Hire_Test.cpp
#include <iostream>
#include <bits/stdc++.h>
using namespace std;


int main()
{
    int T;
    cin >> T;
    while (T--)
    {
        int N, M, X, Y;
        cin >> N >> M >> X >> Y;
        string inp[N];
        for (int i = 0; i < N; i++)
        {
            cin >> inp[i];
        }
        for (int i = 0; i < N; i++)
        {
            int F_Count = count(inp[i].begin(), inp[i].end(), "F");
            int P_Count = count(inp[i].begin(), inp[i].end(), "P");
        }
    }
    return 0;
}

File: Codechef/Hire_Test/Hire_Test.py
T = int(input())
while T:
    T -= 1
    N, M = map(int, input().split())
    X, Y = map(int, input().split())
    inp = []
    for i in range(N):
        ele = input()
        inp.append(ele)
    s = ''
    U_Count, P_Count, F_Count = ele.count("U"), ele.count("O"), ele.count("F")
    if ((F_Count >= X or F_Count >= X - 1) and P_Count >= Y):
        s += '1'
    else:
        s += '0'